builds/MEASLES.md

```diff
@@ -4,12 +4,30 @@
 * This is search fields of `measles[title] AND viruses[filter] AND ("5000"[SLEN] : "20000"[SLEN])`
 * Send to : Complete Record : File : Accession List
 * This downloads the file `sequence.seq`
-* Open this file and remove the `.1`, `.2`, etc... from the accession numbers
+* Remove the `.1`, `.2`, etc... from the accession numbers in `sequence.seq`:
+
+    ```
+    sed -i '' -e 's/.1$//g' -e 's/.2$//g' sequence.seq
```

Comment:
There's a few things not quite right with this sed command.

`.` needs to be escaped to match only a literal `.`, otherwise it matches any character. This means that `-e 's/.1$//g'` turns strings like `43221` into `432`. While the specific content of `sequence.seq` may not include unversioned accessions at the moment, this is corruption waiting to happen.

The double replace (two `-e`'s / separate `s///`) mean that the two operations are always performed, when they should be exclusive or one operation. This means that as is it would turn `43221` into `432` (as above) and then just `4`.

To match `.3`, `.4`, …, `.11`, etc. the single pattern should be something like `s/[.][1-9][0-9]*$//`.

GNU sed errors with `-i ''` because it can't open a file with an empty string.

    sed: can't read : No such file or directory

The correct usage of `-i` / `--in-place` in GNU sed when you don't want a backup file is to just specify the option without a backup suffix at all.